[2023-05-08 05:53:02,144] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dst_report_dags.get_file_from_bucket manual__2023-05-08T05:29:24.071529+00:00 [queued]>
[2023-05-08 05:53:02,155] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dst_report_dags.get_file_from_bucket manual__2023-05-08T05:29:24.071529+00:00 [queued]>
[2023-05-08 05:53:02,157] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-08 05:53:02,159] {taskinstance.py:1377} INFO - Starting attempt 2 of 2
[2023-05-08 05:53:02,160] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-08 05:53:02,177] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_file_from_bucket> on 2023-05-08 05:29:24.071529+00:00
[2023-05-08 05:53:02,182] {standard_task_runner.py:52} INFO - Started process 22 to run task
[2023-05-08 05:53:02,189] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dst_report_dags', 'get_file_from_bucket', 'manual__2023-05-08T05:29:24.071529+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/dst_report_dags.py', '--cfg-path', '/tmp/tmp7qxuxz5i', '--error-file', '/tmp/tmpogc1nz8v']
[2023-05-08 05:53:02,191] {standard_task_runner.py:80} INFO - Job 14: Subtask get_file_from_bucket
[2023-05-08 05:53:02,305] {task_command.py:371} INFO - Running <TaskInstance: dst_report_dags.get_file_from_bucket manual__2023-05-08T05:29:24.071529+00:00 [running]> on host dstreportdagsgetfilefrombucket-56375bbd4c4b450697470ee3d5af00d0
[2023-05-08 05:53:03,396] {logging_mixin.py:115} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/models/renderedtifields.py:249 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2023-05-08 05:53:03,405] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dst_report_dags
AIRFLOW_CTX_TASK_ID=get_file_from_bucket
AIRFLOW_CTX_EXECUTION_DATE=2023-05-08T05:29:24.071529+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-08T05:29:24.071529+00:00
[2023-05-08 05:53:05,653] {python.py:173} INFO - Done. Returned value was: None
[2023-05-08 05:53:05,663] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=dst_report_dags, task_id=get_file_from_bucket, execution_date=20230508T052924, start_date=20230508T055302, end_date=20230508T055305
[2023-05-08 05:53:05,723] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-08 05:53:05,750] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
